Tryouts/Prototypes/Shell/MainWindow.xaml.cs

```diff
@@ -40,13 +41,59 @@ public partial class MainWindow : Window
         private ManifestModel config;
         private ModuleModel[]? modules;
 
+        private string commandLineURL;
+        private string[] hereIsMyCode = (Application.Current as App).CustomerCode;
```

Comment:
If you know the type of Application.Current is always App then use direct cast. If you don't know/can't guarantee that then using `as` is correct, however you should check for null result then.